lib/ffmpegCommandService.js

```diff
@@ -515,28 +519,32 @@ class FfmpegCommandService {
         return {filter, maps};
     }
 
-    _createFilterForTextOverlay(textOverlay, i) {
+    _createFilterForTextOverlay(textOverlay, i, command) {
+        // TODO: doesn't appear to work?
         const bashCharsToEscapeRegex = /!|'|"/g;
 
         // @TODO: add handling for escaping special characters in bash.  Know it's at least ! that needs escaping.
         const text = textOverlay.text.replace(bashCharsToEscapeRegex, "\\$&");
 
-        const defaultMargin = this._fontConfig.defaultMargin || DEFAULT_FONT_MARGIN;
+        // scale the margin with the same ratio as font size
+        let defaultMargin = this._fontConfig.defaultMargin || DEFAULT_FONT_MARGIN;
+        defaultMargin = (16 / defaultMargin) * (command.output.dimensions.height / 16);
 
         const x = this._calculateHorizontalPosition(textOverlay.xLoc, defaultMargin);
         const y = this._calculateVerticalPosition(textOverlay.yLoc, defaultMargin);
+        const size = (16 / textOverlay.fontSize) * (command.output.dimensions.height / 16);
```

Comment:
Same here, I think this can be done with the input vs inside of whipple. That way its the users choice to have font scaling or not.

Reply:
when would a user not want font scaling?

Reply:
recommend adding a tiny comment that explains why `16`

Reply:
@Chuwiey referring to a user being of whipple, not a typical customer/user who is enjoying an end application.

Reply:
I think we could also enable a flag in the output command such as `output.fontScaling = true`

Reply:
fair